### ISSUE TITLE
Create ROADMAP on what projects you're looking for / initiatives

### ISSUE THREAD

**Issue by caniszczyk** [cdfoundation/toc#27]
The @cdfoundation/toc should consider scoping out a basic ROADMAP on what type of projects it is seeking and some mid-long term goals.

**Comment by danlopez00** [comment]
A great example of a trailmap from cncf: https://raw.githubusercontent.com/cncf/trailmap/master/CNCF_TrailMap_latest.png

**Comment by afrittoli** [comment]
@tracymiranda I think some work has been done on this - is there anything we could publish on the ToC repo?

**Comment by tracymiranda** [comment]
Yes the strategy committee had some discussions around project portfolio. The outcomes were high-level:

- cdf projects should embody best practices or help steer folks towards good modern software delivery practices. 
- cdf projects can be any part of the cdf landscape not just pipeline orchestration, see https://landscape.cd.foundation/, see image for brainstormed categories

![CD Foundation - Governing board - Project portfolio - 2021-04-15](https://user-images.githubusercontent.com/5173122/125097408-18c8c000-e0a4-11eb-8a41-11bf26fbf698.png)
 
That's about it, so please feel free to use or not as you see fit.

**Comment by oleg-nenashev** [comment]
@tracymiranda I think it would be a great start if we could publish it somewhere as a draft. Then we could iterate on that at the TOC meetings and in the mailing list, and maybe officially publish it in September after the vacation period

**Comment by oleg-nenashev** [comment]
FYI @tracymiranda @caniszczyk @dlorenc  I have added this topic to the CDF TOC meeting agenda on Tuesday: https://docs.google.com/document/d/1uBHar55fTInWF9Li4t0lyG3tTC8BRLU0FfBfsgk_Jrs/edit?ts=5c9580be#heading=h.isp7qhtdm7dr . I think it would be important to have a consensus in @cdfoundation/toc , and thanks for the proposal above

**Comment by sbtaylor15** [comment]
Moving over to the [cdfoundation/landscape]( https://github.com/cdfoundation/landscape) repo to update the landscape with the missing categories.  PRs in the landscape should be done to add in projects and companies into the appropriate category.
